proto/proto/transaction.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+package penumbra.transaction;
+
+// A Penumbra transaction.
+message Transaction {
+  TransactionBody body = 1;
+  // The binding signature is stored separately from the transaction body that it signs.
```

Comment:
Here and elsewhere the idea is to separate signatures on some structure from the structure itself, so that we can more easily define what data the signature is signing, without having to go field by field.